include/seqan3/io/alignment_file/output.hpp

```diff
@@ -731,16 +791,26 @@ template <detail::Fields    selected_field_ids>
 alignment_file_output(std::filesystem::path &&, selected_field_ids const &)
     -> alignment_file_output<selected_field_ids,
                              typename alignment_file_output<>::valid_formats,
-                             typename alignment_file_output<>::stream_type,
+                             typename alignment_file_output<>::stream_char_type,
+                             ref_info_not_given>;
+
+template <OStream2                  stream_type,
+          AlignmentFileOutputFormat file_format,
+          detail::Fields            selected_field_ids>
+alignment_file_output(stream_type &&, file_format const &, selected_field_ids const &)
+    -> alignment_file_output<selected_field_ids,
+                             type_list<file_format>,
+                             typename std::remove_reference_t<stream_type>::char_type,
                              ref_info_not_given>;
 
-template <OStream<char>             stream_type,
+
+template <OStream2                  stream_type,
```

Comment:
The pending PR #958 inserts documentation for all deduction guides here, so there is a risk of inconsistencies between code and documentation when merging both. We have to keep an eye on this.

Reply:
I added documentation for all as I added some and they start to get confusing. So you can just remove the comments in your PR and it won't cause merge conflicts. :)

Reply:
Cool, thank you!